modules/social_features/social_search/social_search.info.yml

```diff
@@ -15,5 +15,6 @@ dependencies:
   - social:social_editor
   - social:social_event
   - social:social_group
+  - social:social_group_flexible_group
```

Comment:
this is interesting, because flexible groups in 11 is an optional module, why would that be a dependency for this?

Reply:
It is because the `field.storage.group.field_group_allowed_join_method` is a dependency of `search_api.index.social_groups.yml` and this was added in https://git.drupalcode.org/project/search_api/-/blob/8.x-1.x/search_api.post_update.php#L13. And without this dependency, fresh installations were failing.